bundle-size/test/api.test.js

```diff
@@ -164,6 +164,15 @@ describe('bundle-size api', () => {
           'dist/v0/amp-ad-0.1.js': 4.56,
         },
       };
+
+      this.pullRequestFixture = getFixture('pulls.get.19603');
+      nock('https://api.github.com')
+        .persist()
+        .get('/repos/ampproject/amphtml/pulls/19603')
+        // Make the reply a callback function, because nock stringifies the
```

Comment:
Does it have to be a member property? Could we `let pullRequestFixture` at the top of the describe block, then assign to it in beforeEach and access it as a local variable instead?

Reply:
It does not have to be a member, you are correct. I still want to maintain the callback trick though because by default almost all the tests will just use the pre-existing fixture. Done.